scripts/validate_container.sh

```diff
@@ -251,6 +251,9 @@ done
 
 if [ -n "$(find $(pwd) -name '*.ipynb')" ]; then
     echo "Validating notebooks:"
+    echo "Installing packages required for notebook validation"
+    ## Ref: https://github.com/qutip/qutip/issues/2412
+    python3 -m pip install --no-cache-dir "qutip<5" matplotlib
```

Comment:
Instead of installing this here, it makes sense to install it by default in the docker image, specifically along with the notebook package: https://github.com/NVIDIA/cuda-quantum/blob/9dbfec4ec2cc6afb0f0c5657ccffc3f578417e46/docker/release/cudaq.Dockerfile#L65
Then also the arm validation should pass (qutip is failing to install on arm here, since it doesn't have gcc to build it)

Reply:
I took the liberty of updating your branch to confirm that this works.